lib/src/main/java/com/diffplug/spotless/groovy/RemoveSemiColonsStep.java

```diff
@@ -0,0 +1,63 @@
+package com.diffplug.spotless.groovy;
+
+import com.diffplug.spotless.FormatterFunc;
+import com.diffplug.spotless.FormatterStep;
+
+import java.io.BufferedReader;
+import java.io.Serializable;
+import java.io.StringReader;
+
+/**
+ * Removes all semicolons from the end of lines.
+ *
+ * @author Jose Luis Badano
+ */
+public final class RemoveSemiColonsStep {
```

Comment:
We can rename these `RenameSemiColons*` to `RenameSemicolons*`

Reply:
And this?

Reply:
It doesn't make sense for me that name.
The idea of this step is to remove the semicolons at the end of the lines in the groovy files, since they are optional.
If change the name to `RenameSemiColons` it would be harder to understand the purpose of the step by reading only the name.

WDYT?

Reply:
I meant rename all `SemiColons` to `Semicolons`, like `RemoveSemiColonsStep` to `RemoveSemicolonsStep`, cause `semicolon` is a complete word.

https://en.wikipedia.org/wiki/Semicolon

Reply:
Oh!! Yeah, got it. Yep, let me do that change quickly

Reply:
Done -> [5781dea](https://github.com/diffplug/spotless/pull/1881/commits/5781dea7679fb5be6a652fe5d8aa4a9920dc47e8)